### ISSUE TITLE
Android studio open flutter project is abnormal

### ISSUE THREAD

**Issue by shijinliang** [flutter/flutter-intellij#4246]
Yesterday i upgrade flutter 1.12.13+hotfix.5 and android studio 3.5.3, when i open flutter project with android studio, it's abnormal. android studio auto open flutter's android project, but it's error and can't close.(Then I upgraded it to 3.6 rc1, same problem)

![20201229112803](https://user-images.githubusercontent.com/11717694/71552008-6afc4000-2a2e-11ea-89bf-7d26347adcf9.jpg)
![20201229112838](https://user-images.githubusercontent.com/11717694/71552009-6afc4000-2a2e-11ea-8937-f33cb9dff104.jpg)

flutter doctor -v

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.15.2 19C57,
    locale zh-Hans-CN)
    • Flutter version 1.12.13+hotfix.5 at /Users/kamy/flutter
    • Framework revision 27321ebbad (3 weeks ago), 2019-12-10 18:15:01 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/kamy/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = /Users/kamy/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.2.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build
      1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.41.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.7.1

[!] Connected device
    ! No devices available
```

**Comment by devoncarew** [comment]
Moved to flutter/flutter-intellij.

I agree with the comment above - this may not be a flutter issue. That stack trace:

<img width="1114" alt="Screen Shot 2020-01-03 at 8 17 56 AM" src="https://user-images.githubusercontent.com/1269969/71734805-981b6a80-2e01-11ea-9538-c351691b02d3.png">

looks like an issue in Android Studio itself, and not in the Flutter plugin. Can you file a bug there (https://developer.android.com/studio/report-bugs)?

**Comment by stevemessick** [comment]
I'm not sure how the NPE relates but the empty `_android` window is a dup of #4217. I think we can close this one.

**Comment by stevemessick** [comment]
Fixed by #4271
